docs/tutorials/getting_started.rst

```diff
@@ -86,601 +117,85 @@ instance, for Mark 5B files one needs the number of channels used, as well as
-:ref:`Specific file format <specific_file_formats_toc>` documentation.
-
-For the rest of this section, we will stick to VDIF files.
-
-Decoding Data and the Sample File Pointer
------------------------------------------
-
-By giving the openers a ``'rs'`` flag, which is the default, we open files in
-"stream reader" mode, where a file is accessed as if it were a stream of
-samples.  For VDIF, `~baseband.vdif.open` will then return an instance of
-`~baseband.vdif.base.VDIFStreamReader`, which wraps a raw data file with
-methods to decode the binary |data frames| and seek to and read data
-|samples|.  To decode the first 12 samples into a `~numpy.ndarray`, we would
-use the `~baseband.vdif.base.VDIFStreamReader.read` method::
-
-    >>> from baseband import vdif
-    >>> from baseband.data import SAMPLE_VDIF
-    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
-    >>> d = fh.read(12)
-    >>> type(d)
-    <... 'numpy.ndarray'>
-    >>> d.shape
-    (12, 8)
-    >>> d[:, 0].astype(int)    # First thread.
-    array([-1, -1,  3, -1,  1, -1,  3, -1,  1,  3, -1,  1])
-
-As discussed in detail in the :ref:`VDIF section <vdif>`, VDIF files are
-sequences of data frames, each of which is comprised of a :term:`header` (which
-holds information like the time at which the data was taken) and a
-:term:`payload`, or block of data.  Multiple concurrent time streams can be
-stored within a single frame; each of these is called a ":term:`channel`".
-Moreover, groups of channels can be stored over multiple frames, each of which
-is called a ":term:`thread`".  Our sample file is an "8-thread, single-channel
-file" (8 concurrent time streams with 1 stream per frame), and in the example
-above, ``fh.read`` decoded the first 12 samples from all 8 threads, mapping
-thread number to the second axis of the decoded data array.  Reading files with
-multiple threads and channels will produce 3-dimensional arrays.
-
-``fh`` includes ``shape``, ``size`` and ``ndim``, which give the shape, total
-number of elements and dimensionality of the file's entire dataset if it was
-decoded into an array.  The number of |complete samples| - the set of samples
-from all available threads and channels for one point in time - in the file is
-given by the first element in ``shape``::
-
-    >>> fh.shape    # Shape of all data from the file in decoded array form.
+Baseband file objects are wrappers around Python file objects, and have the
+same interface, including `~baseband.vlbi_base.base.VLBIStreamReaderBase.seek`
+for seeking to different parts of the file,
+`~baseband.vlbi_base.base.VLBIStreamReaderBase.tell` for reporting the file
+pointer's current position, and
+`~baseband.vlbi_base.base.VLBIStreamReaderBase.read` for reading data.  The
+main difference is that Baseband file objects read and navigate in units of
+samples.
+
+Let's read some samples from the VDIF file::
+
+    >>> data = fh_vdif.read(3)
+    >>> data[:, 0]      # Print first thread only.
```

Comment:
Just print the whole here! Your explanation about the shape is then a good way to note that one can inspect things by just doing `fh_vdif` or `fh_vdif.info`